### PR TITLE
Task5 7/auth mongo postgres

### PR DESCRIPTION
**Pull request by YahorDauhaliuk**
For toggle database please use config.js

### CODE REVIEW

File: mongo/models/city.model.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import mongoose, { Schema } from 'mongoose';
+
+const CitySchema = new Schema({
+	name: { type: String },
+	country: { type: String },
+	capital: {
+		type: Boolean,
+		required: function () {
```
Comment (abruzgin) [inline mongo/models/city.model.js]: Why so?) Simply `required: true`

Comment (YahorDauhaliuk) [reply mongo/models/city.model.js]: It was a discovery for me - use a function instead of simple boolean) Fixed.

File: routes/auth/logout.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { Router } from 'express';
+import controllers from 'controllers';
+import isAuth from 'middlewares/isAuth';
```
Comment (abruzgin) [inline routes/auth/logout.js]: Also remove here and if `refreshToken` file

File: routes/index.js [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+import city from './city';
+
+import isAuth from 'middlewares/isAuth';
 
 export default (app) => {
-	app.use('/api/products', product);
-	app.use('/api/users', users);
+	app.use('/api/signup', signUp);
+	app.use('/api/signin', signIn);
+	app.use('/api/logout', isAuth, logout);
```
Comment (abruzgin) [inline routes/index.js]: For better view, please, move the routes which doesn't need `isAuth` to one block, then apply `app.use('/api', isAuth)` and then put other routes which need this verification like
```javascript
// routes which don't need verification
app.use('/api/signup', signUp);
................
// verification middleware application
app.use('/api', isAuth);
// verification-demanded routes
app.use('/api/products',product);
```

File: routes/auth/logout.js (hunk 2 above)
Comment (YahorDauhaliuk) [reply routes/auth/logout.js]: Done.

File: routes/index.js (hunk 3 above)
Comment (YahorDauhaliuk) [reply routes/index.js]: Done.

**Pull request merged**